src/modules/Login/action.js

```diff
@@ -139,7 +140,8 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
     accountInitObject.archivedWalletIds = archivedWalletIds
 
     const loadedSyncedSettings = await getSyncedSettings(account)
-    const mergedSyncedSettings = mergeSettings(loadedSyncedSettings, SYNCED_ACCOUNT_DEFAULTS, SYNCED_ACCOUNT_TYPES)
+    const syncedSettings = { ...loadedSyncedSettings } // will prevent mergeSettings trying to find prop of undefined
```

Comment:
So, like `const syncedSettings = loadedSyncedSettings || {}` might be another way to do the same thing.